cypress/integration/basic/observer.spec.ts

```diff
@@ -16,5 +23,26 @@ if (Cypress.env("FLEET_TIER") === "basic") {
 
       // TODO write the test!
     });
+
+    it("Should verify Teams on Hosts page", () => {
+      cy.login("marco@organization.com", "user123#");
+      cy.visit("/hosts/manage");
+  
+      cy.findByText("All hosts which have enrolled in Fleet").should("exist");
+
+      // TODO: can see the "Team" column in the Hosts table
+      // cy.contains(".table-container .data-table__table th", "Team").should("be.visible");
+    });
+
+    it("Should verify hidden items on Hosts page", () => {
+      cy.login("marco@organization.com", "user123#");
+      cy.visit("/hosts/manage");
+  
+      cy.findByText("Packs").should("not.exist");
+      cy.findByText("Packs").should("not.exist");
+
+      // TODO: can see the "Team" column in the Hosts table
+      // cy.contains(".table-container .data-table__table th", "Team").should("be.visible");
+    });
```

Comment:
One slightly different thing I've picked up from the Cypress best practices: https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion

This is the inspiration for the "flow" tests we have here in the cypress directory. Since the initial setup is expensive and best practices recommend making larger tests we've adopted that strategy.

Reply:
Maybe we only need 2 queries to seed since there's only 2 variations of queries needed to test (observer_can_run true vs false)? This might grow in size (algebraic combinations) as we start adding other permissions to queries